src/test/scala/chiselTests/Math.scala

```diff
@@ -0,0 +1,38 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import org.scalacheck.Shrink
+
+class Math extends ChiselPropSpec {
+  import chisel3.util._
+  // Disable shrinking on error.
+  implicit val noShrinkListVal = Shrink[List[Int]](_ => Stream.empty)
+  implicit val noShrinkInt = Shrink[Int](_ => Stream.empty)
+
+  property ("unsignedBitLength is computed correctly") {
+    forAll(safeUIntWidth) { case (width: Int) =>
+      for ( offset <- List(-1, 0, 1)) {
```

Comment:
I'm not sure this test is testing anything since it's basically the duplicating the code used to implement unsignedBitLength. Consider just dropping in a few values and hand-coding the result.

Reply:
Corrected in ec3e030